tcks/apis/connector/docs/pom.xml

```diff
@@ -33,21 +33,13 @@
     <name>connector tck</name>
     <description>Connector docs</description>
 
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
We might also need `<skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>` according to https://github.com/sonatype/nexus-maven-plugins/blob/main/staging/maven-plugin/README.md#plugin-flags.

Reply:
I tested running the nexus-staging:deploy locally and it does skip these, so I thin the current setting in the parent pom that reads the maven.deploy.skip property as the default skipNexusStagingDeployMojo is working.